src/protocol.h

```diff
@@ -268,6 +268,9 @@ enum ServiceFlags : uint64_t {
     // serving the last 288 (2 day) blocks
     // See BIP159 for details on how this is implemented.
     NODE_NETWORK_LIMITED = (1 << 10),
+    // NODE_ENCRYPTED means that the node does accept encrypted communication after BIP151
```

Comment:
"does accept"? Maybe change to "NODE_ENCRYPTED: node accepts encrypted communication, see BIP151"?